### PR TITLE
Add some feature

### PR DESCRIPTION
**Pull request by peteguan**
this is Pete Cheng Guan from ctrip.com add GSLB feature on V21 with the same usage regular as former ones

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,64 +0,0 @@
-# Byte-compiled / optimized / DLL files
```
Comment (mdurrant-b3) [inline .gitignore]: Please don't delete our .gitignore, we need it.

File: acos_client.egg-info/PKG-INFO [hunk 2]
```diff
@@ -0,0 +1,109 @@
+Metadata-Version: 1.1
```
Comment (mdurrant-b3) [inline acos_client.egg-info/PKG-INFO]: Had you left the .gitignore in place, these would have been automatically ignored. :)

File: acos_client/v30/network.py [hunk 3]
```diff
@@ -16,4 +16,34 @@
 
 
 class Network(base.BaseV30):
-    pass
+    #implement by wenjie
```
Comment (mdurrant-b3) [inline acos_client/v30/network.py]: We have vlans, ves, lifs, etc in `v30.vlan` and  `v30.interface`

File: acos_client/v30/session.py [hunk 4]
```diff
@@ -24,8 +34,33 @@ def __init__(self, client, username, password):
 
     @property
     def id(self):
-        if self.session_id is None:
+        deviceIP = self.client.host
```
Comment (mdurrant-b3) [inline acos_client/v30/session.py]: The redis session context needs to be it's own class, else we're breaking the API for  everyone else that uses it.

File: build/lib/acos_client/__init__.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2014,  Doug Wiegley,  A10 Networks.
```
Comment (mdurrant-b3) [inline build/lib/acos_client/__init__.py]: Everything in this build dir needs to go.

**Comment by mdurrant-b3** [comment]
Things that need to be fixed in this PR before we can merge it:
- I want my gitignore file back.
- No replication of files in the build dir or egg info
- The redis session code needs to be refactored into something that doesn't break my existing session interface. I'd highly recommend putting those bits into what ever code you're consuming acos-client with. If that is not possible, I'd be happy to work with you on finding a way to conditionally inject and call the dependency.

**Comment by hthompson-a10** [comment]
PR is stale. Closing for now. Please re-open if required.

**Pull request closed without merging**